pkg/kopia/cli/internal/opts/opts.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2024 The Kanister Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package opts
+
+import "github.com/kanisterio/safecli/command"
+
+var (
+	All           = command.NewOption("--all", true)
+	Delta         = command.NewOption("--delta", true)
+	ShowIdentical = command.NewOption("--show-identical", true)
```

Comment:
These can also be moved to functions below right? They are also of the same format? WDYT?

Reply:
The reason I did it is because we are using the `--all` flag exclusively in the `kopia snapshot list...` command in our current implementation, and it is non-conditional. However, I agree with you and think that we need to convert them into functions for consistency. On the command side, we will simply write `opts.All(true)`, etc.